wouso/games/challenge/forms.py

```diff
@@ -6,7 +6,8 @@ def __init__(self, challenge, data=None):
         super(ChallengeForm, self).__init__(data)
 
         for q in challenge.questions.all():
-            field = forms.MultipleChoiceField(widget=forms.CheckboxSelectMultiple, label=q)
+            CSM = forms.CheckboxSelectMultiple
```

Comment:
@razvanch  if i put label = q on the next line, the line will still have more than 80 characters.